src/build.py

```diff
@@ -202,6 +202,12 @@ def CMakeBinDir():
 NODE_BIN = Executable(os.path.join(WORK_DIR,
                                    NODE_BASE_NAME + NodePlatformName(),
                                    'bin', 'node'))
+NPM_BIN = Executable(os.path.join(WORK_DIR,
+                                  NODE_BASE_NAME + NodePlatformName(),
```

Comment:
maybe we should just refactor the directory part into `NODE_DIR` and then have `NODE_BIN = Executable(os.path.join(NODE_DIR, 'node'))`. Would probably even be fewer lines.

Reply:
Ack